routing/async_router.cpp

```diff
@@ -337,8 +337,12 @@ void AsyncRouter::CalculateRoute()
 
   try
   {
-    LOG(LINFO, ("Calculating the route. checkpoints:", checkpoints, "startDirection:",
-                startDirection, "router name:", router->GetName()));
+    double directDistance = 0;
+    for (auto i1 = begin(a), i2 = i1 + 1; i2 != end(a); ++i1, ++i2)
```

Comment:
What is "a" here? Did you compile it?
I don't understand, we print the total route distance 2 times in logs already.